jaxrs_client_utils/src/test/java/com/yahoo/vespa/jaxrs/client/RetryingJaxRsStrategyTest.java

```diff
@@ -111,14 +112,24 @@ public void testRetryLoopsOverAvailableServers() throws Exception {
     }
 
     @Test
-    public void testRetryGivesUpAfterTwoLoopsOverAvailableServers() throws Exception {
-        when(mockApi.doSomething())
-                .thenThrow(new ProcessingException("Fake timeout 1 induced by test"))
-                .thenThrow(new ProcessingException("Fake timeout 2 induced by test"))
-                .thenThrow(new ProcessingException("Fake timeout 3 induced by test"))
-                .thenThrow(new ProcessingException("Fake timeout 4 induced by test"))
-                .thenThrow(new ProcessingException("Fake timeout 5 induced by test"))
-                .thenThrow(new ProcessingException("Fake timeout 6 induced by test"));
+    public void testRetryGivesUpAfterOneLoopOverAvailableServers() {
+        jaxRsStrategy.setMaxIterations(1);
+        testRetryGivesUpAfterXIterations(1);
+    }
+
+    @Test
+    public void testRetryGivesUpAfterTwoLoopsOverAvailableServers() {
+        testRetryGivesUpAfterXIterations(2);
```

Comment:
Should `jaxRsStrategy.setMaxIterations(2);` be called prior to this, or is it OK to depend on the default value here?

Reply:
I wanted to test the default, which should still be 2.